public/docs/_examples/tutorial/dart/pubspec.yaml

```diff
@@ -0,0 +1,15 @@
+name: angular2_tour_of_heroes
+version: 0.0.1
+environment:
+  sdk: '>=1.13.0 <2.0.0'
```

Comment:
I think we should. Only for using caret notation in dependencies, we need 1.6.0 or higher I think. For async/await 1.9.0 or higher. Null aware operator 1.12.0 or higher (which I use). But to be safe, I always use the latest stable version.